experiments/functional/snapshot_deletion/jiva/test_replica_restart.yml

```diff
@@ -79,6 +88,12 @@
    - name: Getting the master replica pod name corresponding to the IP
      shell: kubectl get pod -l openebs.io/replica=jiva-replica,openebs.io/persistent-volume={{ pv_name }} -n {{ operator_ns }} --no-headers -o jsonpath='{.items[?(@.status.podIP=="{{ replica_ip }}")].metadata.name}'
      register: replica_name
+     when: storage_class != 'jiva-csi-sc'
+
+   - name: Getting the master replica pod name corresponding to the IP
```

Comment:
@payes  this task is duplicate of the previous task.

Reply:
This is for CSI-based volumes and the previous task was for non-csi based volumes.
```storage_class != 'jiva-csi-sc'```